rho/vault.py

```diff
@@ -23,6 +23,8 @@
 from rho.translation import _ as t
 
 PROMPT = "Please enter your rho vault password: "
+ERROR_PROMPT = 'Error: The vault password cannot be empty. '\
+    'Please provide a none empty password.'
```

Comment:
I am not sure but I think the message should be "Please provide a non empty password", mind non instead of none?

Reply:
yep, I'll fix it. should be `non-empty`.